trimmer_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    length = @nameable.correct_name.length
+    if length <= 10
+      @nameable.correct_name
+    else
+      @nameable.correct_name[0..10]
```

Comment:
- [ ] The `corect_name` function in the _Trimmer Decorator_ should return the given name with a maximum of **10** characters. If it's longer it should trim the word. Your  `corect_name` function is however, returning characters more than 10. Kindly ensure to improve upon the same.

**Characters more than 10 returned:**
![image](https://user-images.githubusercontent.com/12745474/210604384-eb13e294-f08a-418e-a986-a3a581bcd681.png)